### DISCUSSION TITLE
New to SO (Splunk Before), Struggling with missing dashboards in Elastic

### DISCUSSION THREAD

**Discussion by pastorlibre** [Security-Onion-Solutions/securityonion#11413] [2.4]
### Version

2.4.1

### Installation Method

Other (please provide detail below)

### Description

configuration

### Installation Type

Standalone

### Location

cloud

### Hardware Specs

Exceeds minimum requirements

### CPU

24

### RAM

32gb

### Storage for /

500

### Storage for /nsm

2tb

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi, I am very new to Security Onion, and its a struggle coming from Splunk. My question is I installed the Azure Logs in Elastic (out of SO) and it is pulling data but the dashboards (13 is what it should have) is missing. Says Dashboards can not be found. I can find any resources or knowledge on how to get them, check them, or anything. Can anyone point me in a direction. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by defensivedepth** [comment]
To confirm my understanding:

You have installed the Elastic Agent `Azure Logs` Integration?

After installing the integration (without configuring the Azure connection info), I still seem to be able to pull up the dashboards albeit without data on them.

Are you getting a 404 error? Can you post a screenshot of what you are seeing?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/954732/58ccd17a-a08c-4085-ad34-5b5e491a0af9)
